osquery/core/system.cpp

```diff
@@ -94,20 +94,21 @@ struct tm* localtime_r(time_t* t, struct tm* result) {
   _localtime64_s(result, t);
   return result;
 }
+
 #endif
 
 std::string getHostname() {
 #ifdef WIN32
-  long size = 256;
+  long size = MAX_PATH;
```

Comment:
If `MAX_PATH` exists on windows can this logic be expressed without preprocessor conditionals?